lemur/auth/service.py

```diff
@@ -11,10 +11,13 @@
 import jwt
 import json
 import binascii
+import logging
 
 from functools import wraps
 from datetime import datetime, timedelta
 
+from dd_internal_authentication.jwt_authenticator import JWTAuthenticator
```

Comment:
Could we verify the JWT using pyjwt and existing Lemur dependencies? https://github.com/Netflix/lemur/blob/bf5978cea99ec52451f7a2c1b98f51d7b51a6de5/lemur/auth/views.py#L108

This definitely looks reasonable, but it's somewhat undesirable to leak internal libs like this since this is an open source project. It would make it really tough to contribute back changes from our fork.

Reply:
I think we can but we will end up just copying the core code from this package to achieve it.  If that's more desirable it can definitely be done, but is there any benefit to using this package wrt hiding from the general public exactly how we validate the JWT?

Reply:
I don't think we really want to be bringing in DD-specific dependencies into this project if we can avoid it. I don't see much benefits to hiding a JWT verification implementation, and only see downsides in terms of complexity and exposing unnecessary internal details of our infrastructure publicly. Lemur already has capability to verify JWTs that could be re-used, there should in theory not be much copying required.